packages/devtools_app/test/treemap_test.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/src/charts/treemap.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'support/utils.dart';
+import 'support/wrappers.dart';
+
+void main() {
+  group('Treemap', () {
+    TreemapNode root;
+
+    setUp(() async {
+      root = await loadInstructionSizesJson();
+    });
+
+    void changeRoot(TreemapNode newRoot) {
```

Comment:
@kenzieschmoll This is also a duplicate that exists in `CodeSizeController`. Should I be taking it out to a different file or is it okay if I keep both?

Reply:
i don't think you actually need this method. just pass a null or noop callback ( `() {}` ) for onRootChangedCallback

Reply:
I think if we want to test what happens after zooming in we need the function